applications/openshift/scc/scc_limit_container_allowed_capabilities/rule.yml

```diff
@@ -44,13 +44,13 @@ ocil: |-
 
 warnings:
 - general: |-
-    {{{ openshift_filtered_cluster_setting({'/apis/security.openshift.io/v1/securitycontextconstraints': '[.items[] | select(.metadata.name != "privileged")] | map(.allowedCapabilities == null)'}) | indent(4) }}}
+    {{{ openshift_filtered_cluster_setting({'/apis/security.openshift.io/v1/securitycontextconstraints': '[.items[] | select(.metadata.name != "privileged") | select(.metadata.name | test("^.*-v2$"; "") | not)] | map(.allowedCapabilities == null)'}) | indent(4) }}}
```

Comment:
Wondering if we exclude all SCCs ending with v2, will it give false-positive result

Reply:
right, that was one of the things I was considering. And you are right that v2 is too generic to use for a filter (what if someone has `myapp-v2` SCC?).
We can filter out the names of the SCCs ourselves, but a chain of selects would be ugly. I'll try to come up with a regex instead that I can use in `test`.
Is there anything nicer we could use?

Reply:
If we chained all the SCC together using selects, we'd need to update that list each time it changed, right?

Reply:
Unfortunately our SCC annotations are not super consistent, most system SCCs have `include.release.openshift.io/self-managed-high-availability` but not all (e.g. not hostmounter). Most have `release.openshift.io/create-only` but not all.

So in the end, I think we could add a list.

@Vincent056 do you know if we can use a variable in the JQ filter? If yes, then it would be nice to make the list of SCCs that are OK to allow capabilities as a variable that would default to those that we ship and the customers could add their own.

Reply:
sorry for that late reply, yes, I think we can use the variable in the JQ filter, you just need to create a variable and use it as {{.var_scc_exclude_name}}

Reply:
thank you, let me work on that

Reply:
Done. @Vincent056 I have one more question, do we need to document the variable somehow?

Reply:
actually, hold the review, I'm going to add the docs either way